src/lightstep-opentelemetry-node.ts

```diff
@@ -0,0 +1,163 @@
+import {
+  ConsoleLogger,
+  LogLevel,
+  B3Propagator,
+  CompositePropagator,
+  HttpCorrelationContext,
+  HttpTraceContext,
+} from '@opentelemetry/core';
+import { HttpTextPropagator, Logger } from '@opentelemetry/api';
+import { NodeSDK } from '@opentelemetry/sdk-node';
+import {
+  LightstepNodeSDKConfiguration,
+  LightstepConfigurationError,
+} from './types';
+import {
+  CollectorTraceExporter,
+  CollectorProtocolNode,
+} from '@opentelemetry/exporter-collector';
+
+const OPTION_ALIAS_MAP: { [key: string]: string } = {
+  LS_ACCESS_TOKEN: 'token',
+  LS_SERVICE_NAME: 'serviceName',
+  OTEL_EXPORTER_OTLP_SPAN_ENDPOINT: 'spanEndpoint',
+  OTEL_PROPAGATORS: 'propagators',
+};
+
+const DEFAULTS: Partial<LightstepNodeSDKConfiguration> = {
+  spanEndpoint: 'https://ingest.lightstep.com:443/api/v2/otel/trace',
```

Comment:
I guess we will have `metricsEndpoint` in future, should this be a plural then ?

Reply:
The names here and in https://github.com/lightstep/opentelemetry-node/pull/1#discussion_r455338710 are defined by our internal document. Each different locl library should agree to use the same names.